src/components/SignUpForm/SignUpForm.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState, FormEvent } from 'react';
+import './SignUpForm.css';
+import Button from '../Button/Button';
+
+function SignUpForm() {
+	const [username, setUsername] = useState('');
+	const [email, setEmail] = useState('');
+	const [password, setPassword] = useState('');
+
+	async function handleSignUp(event: FormEvent) {
+		event.preventDefault();
+		const response = await fetch('http://localhost:3000/sign-up', {
+			method: 'POST',
+			headers: {
+				'Content-Type': 'application/json',
+			},
+			body: JSON.stringify({ username, email, password }),
+		});
+
+		if (response.ok) {
+			const data = await response.json();
+			// Test log to see the response after sign up attempt
+			console.log('Sign up successful:', data);
```

Comment:
Weirdly I'm not seeing a success message in the console but the data is appearing in the db.

Reply:
Never mind, I see it in the browser! Sorry about that.